### DISCUSSION TITLE
Pull request for time series

### DISCUSSION THREAD

**Discussion by Adriannaluger** [earthlab-education/Earth-Analytics-2023-01-Intro#324] [! GitHub help]
Hi! I am having some trouble completing a pull request for my time series. Could this be because my temperature pull request hasnt been completed? 
There is no 'compare & pull request' button for me available even though my url for my time series html is in the earthlab analysis tab (under my branch and in my porfoloio). 
Thanks!! :)

**Comment by eculler** [comment]
@Adriannaluger did this get worked out? It is true that if your temperature pull request hasn't yet been merged, any further changes you make to the `Earth-Analytics-2023-01-Intro` repository in your branch will get automatically added to your existing PR.

**Reply by pth6570** [reply]
@eculler I'm having the same issue for the current assignment. It looks like my streamflow pull request hasn't been approved yet and that my addition of the raster assignment has been added to that pull request. Is there anything else I need to do? Or should I just wait for approval on my existing PR? Thanks!

**Reply by eculler** [reply]
Yeah, I've been waiting for reviews for a bunch of folks, but I should go in and bypass them so we can move on. Don't worry about waiting for me to make your new changes - I'll get it worked out.

**Comment by Adriannaluger** [comment]
@eculler I am actually unsure now. I think they might have got automatically added like you said because I never started a pull request for my time-series yet. However, it appears like it got merged on the main branch of the earth analytics analysis page. Let me know what you see on your end if I need to make any changes. Thanks!
